Mathlib/SetTheory/Ordinal/FixedPoint.lean

```diff
@@ -477,6 +477,19 @@ theorem deriv_eq_enumOrd (H : IsNormal f) : deriv f = enumOrd (Function.fixedPoi
 theorem deriv_eq_id_of_nfp_eq_id {f : Ordinal → Ordinal} (h : nfp f = id) : deriv f = id :=
   (IsNormal.eq_iff_zero_and_succ (deriv_isNormal _) IsNormal.refl).2 <| by simp [h]
 
+@[simp]
+theorem nfp_zero : nfp 0 = id := by
```

Comment:
Can we also have `nfp_zero_left : nfp 0 a = a`? This is the more useful lemma for rewriting